bitrise.yml

```diff
@@ -21,17 +21,15 @@ workflows:
     - gradle-runner:
         title: Run Android Unit Tests
         inputs:
-        - gradle_task: ':test'
+        - gradle_task: ":test"
   primary-send-slack:
     steps:
     - slack:
         inputs:
-        - channel: $SLACK_SDK_CHANNEL
-        - message: >-
-            Build #<$BITRISE_BUILD_URL|$BITRISE_BUILD_NUMBER> of
-            $BITRISE_APP_TITLE@$BITRISE_GIT_BRANCH by
-            $GIT_CLONE_COMMIT_AUTHOR_NAME passed.
-        - webhook_url: $SLACK_WEBHOOK_URL
+        - channel: "$SLACK_SDK_CHANNEL"
+        - message: 'Build #<$BITRISE_BUILD_URL|$BITRISE_BUILD_NUMBER> of $BITRISE_APP_TITLE@$BITRISE_GIT_BRANCH
+            by $GIT_CLONE_COMMIT_AUTHOR_NAME passed.'
```

Comment:
I'm not sure how exactly you'd like to get this text formatted, but I have a feeling that this might be what you're looking for:

```yml
- message: |
  Build "#<$BITRISE_BUILD_URL|$BITRISE_BUILD_NUMBER>" of
  "$BITRISE_APP_TITLE@$BITRISE_GIT_BRANCH" by
  "$GIT_CLONE_COMMIT_AUTHOR_NAME" passed.
```

https://yaml-multiline.info/

Reply:
The output in Slack looks fine to me as is. As long as Bitrise is happy to run it I'm not too worried 😄.